README.md

```diff
@@ -55,6 +55,7 @@ Supported keys of the map are:
 ## License
 
 Copyright © 2016 Coda Hale
+Copyright © 2017 Sentry
```

Comment:
To be extra clear: I didn't mean to imply anything by the copyright. It's a force of habit and I got ahead of myself.

I'd love to discuss your thoughts on the project. I'm more than happy to do whatever you prefer. I don't want to point our users your direction without you knowing well in advance. 😬